attributes/default.rb

```diff
@@ -12,7 +12,7 @@
 default['hashicorp-vault']['service_user'] = 'vault'
 default['hashicorp-vault']['service_group'] = 'vault'
 
-default['hashicorp-vault']['version'] = '0.8.3'
+default['hashicorp-vault']['version'] = '0.9.4'
```

Comment:
You need to add this version into the [vault_installation_binary.rb file](https://github.com/johnbellone/vault-cookbook/blob/master/libraries/vault_installation_binary.rb#L124) with the checksum.

Reply:
After that fix merge conflict and we'll release! Thanks for the help!

Reply:
@johnbellone Hi John, it's already in there. GitHub collapses the diff because the changeset is so large.

I removed the checksums for versions < 0.7.0 because the new config options the cookbook writes out are not supported by versions prior to 0.7.0.

Reply:
@johnbellone I've fixed the merge conflict.